src/Reflection/StringCast/ReflectionClassConstantStringCast.php

```diff
@@ -18,14 +17,13 @@ final class ReflectionClassConstantStringCast
     public static function toString(ReflectionClassConstant $constantReflection) : string
     {
         $value = $constantReflection->getValue();
-        assert($value === null || is_scalar($value));
 
         return sprintf(
             "Constant [ %s %s %s ] { %s }\n",
             self::visibilityToString($constantReflection),
             gettype($value),
             $constantReflection->getName(),
-            (string) $value
+            is_array($value) ? 'Array' : (string) $value
```

Comment:
Mentioned elsewhere: we need a test verifying https://3v4l.org/q8nvd

Reply:
@Ocramius The test exists already: https://github.com/Roave/BetterReflection/blob/master/test/unit/Reflection/StringCast/ReflectionClassConstantStringCastTest.php#L35

Reply:
Ah, interesting! Thanks for hunting it down: guess the error handler was not being tripped :-)